tools/deploy_documentation_tag.sh

```diff
@@ -24,9 +24,9 @@ echo "show current dir: "
 pwd
 
 CURRENT_TAG=`git describe --abbrev=0`
-IFS='.'
-read -ra VERSION <<< "$CURRENT_TAG"
-STABLE_VERSION=`echo $VERSION | cut -d "." -f -2`
+IFS=. read -ra VERSION <<< "$CURRENT_TAG"
+# The "minor" version, so in "0.29.2" it's "29".
+STABLE_VERSION=${VERSION[1]}
```

Comment:
This needs to be:

```suggestion
STABLE_VERSION=${VERSION[0]}.${VERSION[1]}
```

I think because the publish url is `stable/0.33`. basically we want to publish stable versions at: qiskit.org/documentation/stable/major.minor

Reply:
Thanks - I misread the `cut` command. I thought it was only picking out field `2`, as opposed to `-2`.  Fixed in 8534ac9cc.

Reply:
And apparently didn't stop to think that that was kind of weird for a URL.